mobile/lib/common/amount_text_input_form_field.dart

```diff
@@ -2,56 +2,72 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 
 import 'domain/model.dart';
+import 'modal_bottom_sheet_info.dart';
 
-class AmountInputField extends StatelessWidget {
+class AmountInputField extends StatefulWidget {
   const AmountInputField(
       {super.key,
       required this.controller,
       required this.label,
       required this.hint,
       required this.onChanged,
       required this.value,
+      required this.isLoading,
+      this.infoText,
       this.validator});
 
   final Amount value;
   final TextEditingController controller;
   final String label;
   final String hint;
+  final String? infoText;
```

Comment:
Could we use the nullness of infoText as a proxy for whether it has finished loading? I.e, it is null until it has loaded, so we can get rid of the separate tracker bool

Reply:
I thought about it first, but then decided against it because `loading` could be generally useful.